### PR TITLE
Mention the need to use namespace import of code example is used outside of initial namespace

### PR DESCRIPTION
**Pull request by WebHelpersPau**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | Missing inclusions on example 
| Fixed ticket? | N/A

### CODE REVIEW

File: development/architecture/migration-guide/forms/CRUD-forms.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ In the example below, you can see a `ContactFormDataProvider` that queries the d
 namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider;
 
 use Contact;
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider\FormDataProviderInterface;
```
Comment (matks) [inline development/architecture/migration-guide/forms/CRUD-forms.md]: I dont think this is necessary. Classes and interfaces that are in the same namespace `PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider` are automatically found if I am not wrong?

Comment (WebHelpersPau) [reply development/architecture/migration-guide/forms/CRUD-forms.md]: Not on mine, I've been facing it in a module creation...

Comment (matks) [reply development/architecture/migration-guide/forms/CRUD-forms.md]: Hello @WebHelpersPau and apologies for late answer.

I think if you use the above code in the namespace `PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider` you do not need this `use`.

But you used this code in a module creation, which means the namespace was not the same which means the `use` was needed.

So in order to address both situations I will add a mention in the doc ;) so we can have both solutions that fit well with the need.

Comment (matks) [reply development/architecture/migration-guide/forms/CRUD-forms.md]: See https://github.com/PrestaShop/docs/pull/1272/commits/a9f795465169a0bf318bfccd4a0e9be278ad80c0

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @WebHelpersPau

**Comment by matks** [comment]
Backported to 8.x https://github.com/PrestaShop/docs/pull/1309

**Pull request merged**